### PR TITLE
Adding the ability to steer the dream move block and a lot of customizable colors

### PR DESCRIPTION
**Pull request by Vldad**
- Adding a lot of customizable colors and the ability to steer the dream move block
- Only on Lönn for now

### CODE REVIEW

File: Loenn/entities/dream_blocks/dream_move_block.lua [hunk 1]
```diff
@@ -43,11 +67,28 @@ dreamMoveBlock.placements = {
...
+            noCollide = false,
+			canSteer = false,
+			idleButtonsColor = "FFFFFF",
+			movingButtonsColor = "FFFFFF",
+			idleArrowColor = "FFFFFF",
+			movingArrowColor = "FFFFFF",
+			idleWobbleLinesColor = "FFFFFF",
+			movingWobbleLinesColor = "FFFFFF",
+			breakingWobbleLinesColor = "FFFFFF",
+			breakingCrossColor = "FFFFFF",
```
Comment (catapillie) [inline Loenn/entities/dream_blocks/dream_move_block.lua]: indentation
```suggestion
            noCollide = false,
	    canSteer = false,
	    idleButtonsColor = "FFFFFF",
	    movingButtonsColor = "FFFFFF",
	    idleArrowColor = "FFFFFF",
	    movingArrowColor = "FFFFFF",
	    idleWobbleLinesColor = "FFFFFF",
	    movingWobbleLinesColor = "FFFFFF",
	    breakingWobbleLinesColor = "FFFFFF",
	    breakingCrossColor = "FFFFFF",
```

File: Loenn/lang/en_gb.lang [hunk 2]
```diff
@@ -14,6 +14,12 @@ entities.CommunalHelper/DreamJellyfish.attributes.description.tutorial=Whether t
 # Dream Booster
 entities.CommunalHelper/DreamBoosterAny.placements.name.dream_booster=Dream Booster (Any Direction)
 
+#Dream Spring
+entities.CommunalHelper/DreamSpringUp.placements.name.up=Dream Spring (Up)
+entities.CommunalHelper/DreamSpringRight.placements.name.right=Dream Spring (Right)
+entities.CommunalHelper/DreamSpringLeft.placements.name.left=Dream Spring (Left)
+entities.CommunalHelper/DreamSpringDown.placements.name.down=Dream Spring (Down)
+
```
Comment (catapillie) [inline Loenn/lang/en_gb.lang]: ???

File: src/Entities/DreamBlocks/DreamMoveBlock.cs [hunk 3]
```diff
@@ -56,8 +63,28 @@ public enum MovementState
 
     private Player noSquish;
 
+    private List<Image> topButton = new List<Image>();
+
+    private List<Image> leftButton = new List<Image>();
+
+    private List<Image> rightButton = new List<Image>();
```
Comment (catapillie) [inline src/Entities/DreamBlocks/DreamMoveBlock.cs]: nitpick: spacing & syntax
```suggestion
    private List<Image> topButton = new();
    private List<Image> leftButton = new();
    private List<Image> rightButton = new();
```

**Review by catapillie — CHANGES_REQUESTED** [review]

File: Loenn/entities/dream_blocks/dream_move_block.lua (hunk 1 above)
Comment (Vldad) [reply Loenn/entities/dream_blocks/dream_move_block.lua]: Oh I didn't pay attention that the indentation was done with blank spaces. I used tabs which, in my editor didn't show any difference but looks like it did in your own. I'm replacing tabs with 4 spaces then.

File: Loenn/lang/en_gb.lang (hunk 2 above)
Comment (Vldad) [reply Loenn/lang/en_gb.lang]: It was an idea I started developing but it wasn't supposed to ship in this PR, i'm removing it

**Review by catapillie — COMMENTED** [review]
Thanks for the fixes. Will test out later this week hopefully (quite busy atm).

**Review by catapillie — APPROVED** [review]

**Comment by catapillie** [comment]
LGTM, thanks! Fixed some conflict, merging now

**Pull request merged**